packages/ffe-webfonts/sb1-fonts.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
```

Comment:
Den burde ikke være med. Vi glemte å legge inn den i `.gitignore`

Reply:
Jag kan fikse

Reply:
hmm forsatt med skall se

Reply:
Ble fikset i 3b12756 ;-)

Reply:
> > Jo det gjør det helt klart. Tenkte att man kunde få mer ting her som className og andre attributer hvis man neon gang skulle ønske det. Kan gott gå før kun en ariaLabel alltså. Det var før økad flexibilitet men er det ikke intressant så kan vi gå kun før aria-label
> 
> Det blir litt mye «just in case» for min smak. Teknisk sett er denne bra, men jeg vi klarer oss kanskje uten denne fleksibilitet. Kan du redusere det til kun `aria-label`?

Selvsagt kan jag det.